src/ert/config/observations.py

```diff
@@ -417,6 +417,11 @@ def _create_gen_obs(
         else:
             indices = np.arange(len(values))
         std_scaling = np.full(len(values), 1.0)
+        if len({len(stds), len(values), len(indices)}) != 1:
+            raise ValueError(
```

Comment:
Probably want to use ObservationConfigError.with_context and use the filename as context.

Reply:
Updated :+1: